packages/components/src/stepper/style.scss

```diff
@@ -15,21 +15,24 @@
 		display: inline-flex;
 		align-items: center;
 		padding: $gap-small;
-		font-weight: 600;
+		font-weight: 400;
+		color: $muriel-gray-900;
 
 		svg {
 			display: none;
 		}
 
 		&.is-active,
 		&.is-complete {
-			color: $muriel-active;
-
-			.woocommerce-stepper__step-icon {
+			.woocommerce-stepper__step-icon, .components-spinner {
 				background: $muriel-active;
 			}
 		}
 
+		&.is-active {
+			font-weight: 600;
```

Comment:
Not a blocker, but the horizontal stepper does "bounce" a bit when the font-weight shifts from step to step:

<img width="738" alt="Screen Shot 2019-05-20 at 6 18 06 PM" src="https://user-images.githubusercontent.com/10561050/58014553-b743e100-7b2b-11e9-8b70-f8218839b0a0.png">